config/development.json

```diff
@@ -5,16 +5,59 @@
   "filePath": "data/podcasts.json",
   "reviewsPath": "data/reviews.json",
   "cacheExpiration": 1200,
-  "isCacheEnabled": false,
+  "isCacheEnabled": true,
+  "static": "react-client/src",
   "redis": {
     "port": 6379,
     "host": "localhost",
     "cacheDuration": 5
   },
-  "db": {
-    "port": 3306,
+  "cache": [
```

Comment:
please put all of cache configuration under cache key like so:
`"cache":{
cacheExpiration:"",
isCacheEnabled:"",
cacheClearMap:[]
}`